test/settingsMapper.spec.tsx

```diff
@@ -19,7 +21,7 @@ describe('Settings mapper unit tests', () => {
           return 'also works!';
         }
       };
-      const result = settingsMapper.getSettings(initial);
+      const result: {[key: string]: any} = settingsMapper.getSettings(initial);
```

Comment:
`getSettings` returns type `Handsontable.DefaultSettings`. Why we're rewriting it to `{[key: string]: any}` ?

Reply:
Because the `handsontable.d.ts` requires that the callbacks to hooks have the same amount of arguments as the hook itself, e.g.:
```
afterChange?: (changes: [number, string | number, any, any][], source: string) => void;
```

This test case is only checking if they are present and working (they don't require any arguments, `afterChange: () => { return 'works!'; }`), so to simplify the process I've changed the typing on this one.

I can provide some mocked arguments, I guess... What do you think, @wojciechczerniak?

Reply:
👍